cli/exchange/user.go

```diff
@@ -76,6 +94,35 @@ func UserSetAdmin(org, userPwCreds, user string, isAdmin bool) {
 	cliutils.ExchangePutPost("Exchange", http.MethodPatch, cliutils.GetExchangeUrl(), "orgs/"+org+"/users/"+user, cliutils.OrgAndCreds(org, userPwCreds), []int{201}, patchUserReq, nil)
 }
 
+func UserSetHubAdmin(org, userPwCreds, user string, isHubAdmin bool) {
+	// get message printer
+	msgPrinter := i18n.GetMessagePrinter()
+
+	_, userId := cliutils.TrimOrg(org, user)
+
+	// Get the user from the exchange
+	var userList ExchangeUsers
+	httpCode := cliutils.ExchangeGet("Exchange", cliutils.GetExchangeUrl(), "orgs/root/users/"+userId, cliutils.OrgAndCreds(org, userPwCreds), []int{200, 404}, &userList)
```

Comment:
For the case where the exchange API returns an authorization problem, does the user get a reasonable message. I think this case might be somewhat special because the caller MUST be the exchange root user or a hubadmin user, so there should be a message to this effect to help the user.

Reply:
Changed to explain the issue when a 403 is returned